test/isolation/expected/insert_dropchunks_race.out

```diff
@@ -0,0 +1,16 @@
+Parsed test spec with 2 sessions
+
+starting permutation: s1a s2a s1b s2b s1c
+step s1a: INSERT INTO conditions VALUES ('2020-01-03 10:30', 3, 33.4);
+step s2a: SELECT * FROM drop_chunks('conditions', older_than => '2020-03-01'::timestamptz); <waiting ...>
+step s1b: COMMIT;
+step s2a: <... completed>
+drop_chunks    
+
+_timescaledb_internal._hyper_7_187_chunk
+step s2b: COMMIT;
+step s1c: SELECT * FROM _timescaledb_catalog.chunk_constraint LEFT JOIN _timescaledb_catalog.dimension_slice sl ON dimension_slice_id = sl.id;
+chunk_id       dimension_slice_idconstraint_namehypertable_constraint_nameid             dimension_id   range_start    range_end      
+
+188            187            constraint_187                187            7              15779232000000001578528000000000
```

Comment:
this will result in flaky tests, you need to get rid of the counter ids here otherwise reordering or skipping isolation tests will produce different output

you can test this with `make isolationcheck TESTS=insert_dropchunks_race`

Reply:
Counted the number of rows instead. It will give `1` before the commit and `0` after. Check the individual commits to see how it behaves.